docs/tls.md

```diff
@@ -76,10 +80,34 @@ spec:
     name: selfsigning-issuer
 ```
 
-Finally configure MinIO to use the newly created TLS certificate:
+Finally, configure MinIO to use the newly created TLS certificate:
 
 ```yaml
   externalCertSecret:
     - name: tls-minio
       type: kubernetes.io/tls
 ```
+
+---
+
+## Rotate MinIO Operator TLS certificates
+
+In case the operator TLS certificate expired you can just delete the `operator-tls` secret in the `minio-operator` namespace and
+operator will generate a new one automatically, additionally it will copy the `public.crt` portion of the certificate into
+each tenant namespace
+
+---
+
+## Using your own certificate for MinIO Operator
```

Comment:
@Alevsk assuming once I do this, the same behavior aboe applies where operator auto-copies the public.crt over to each tenant

Reply:
@ravindk89 no, for that each tenant has the `externalCaCertSecret` in which the user can pass an array of secrets

Reply:
OK - so if the Operator rolls with custom CA, the user is responsible for passing that CA to `externalCACertSecret`.

We should include that  requirement in this section.